src/EventTickets/resources/admin/components/EventDetailsPage/EventSection/index.tsx

```diff
@@ -47,6 +47,7 @@ export default function EventSection({setUpdateErrors}) {
                 tableHeaders={tableHeaders}
                 data={formattedData}
                 rowActions={EventSectionRowActions({event: data, openEditModal: openModal})}
+                rowActionsAlwaysVisible={true}
```

Comment:
Does this need a specific prop as a first class feature? With the underlying implementation being a CSS rule, would it be better to just target the table by ID and apply CSS?

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7289/commits/8cd2869a9a1e564c98e7257ad6fd6adf4d11242d